setup.py

```diff
@@ -35,7 +35,7 @@
         'configparser; python_version<"3.0"',
         'future>=0.14.0; python_version<"3"',
         'backports.functools_lru_cache; python_version<"3.2"',
-        'jedi>=0.14.1,<0.16',
+        'jedi>=0.16.0',
```

Comment:
Please change this to

    'jedi>=0.17.0,<0.18.0'

because:

* Jedi 0.16 was giving errors for Pandas.
* Jedi is changing a lot lately, so before it reaches 1.0, we don't want to have a `>=` on it here.

Reply:
Done. The CI failures don't look related.

Reply:
Ooops.

A comma was missing between jedi and json-rpcserver line.